src/mokka/src/main/java/pl/hycom/mokka/stubbing/WireMockContext.java

```diff
@@ -0,0 +1,28 @@
+package pl.hycom.mokka.stubbing;
```

Comment:
We may follow WireMock's convention and use 'responsetemplating` package name (under `stubbing`)